pkg/terminal/command.go

```diff
@@ -2845,12 +2845,25 @@ func printBreakpointInfo(t *Term, th *api.Thread, tracepointOnNewline bool) {
 	}
 }
 
+var (
+	depth = make(map[int64]int)
+)
+
 func printTracepoint(t *Term, th *api.Thread, bpname string, fn *api.Function, args string, hasReturnValue bool) {
 	if t.conf.TraceShowTimestamp {
 		fmt.Fprintf(t.stdout, "%s ", time.Now().Format(time.RFC3339Nano))
 	}
 
 	if th.Breakpoint.Tracepoint {
+		idx := strings.Index(fn.Name(), "runtime.morestack_noctxt")
+		// Treat morestack_noctxt as a special case
```

Comment:
Please run `gofmt` on the project sources.

Reply:
As far as I can tell this affects all tracepoints, including the ones set with the trace command. Even so it's wrong because depth is not decremented with recovered panics. I think it would be better to ask for a stacktrace with the tracepoint (there's an option to do that) and try to determine the depth from that.

Reply:
Implemented computation of depth based on stacktrace because of this case